ttnn/cpp/ttnn/operations/eltwise/binary/binary.hpp

```diff
@@ -63,7 +63,8 @@ struct ExecuteBinary {
         const std::optional<const DataType> &output_dtype = std::nullopt,
         const std::optional<MemoryConfig> &memory_config = std::nullopt,
         std::optional<Tensor> optional_output_tensor = std::nullopt,
-        std::optional<FusedActivations> activations = std::nullopt) {
+        std::optional<FusedActivations> activations = std::nullopt,
+        std::optional<const std::string> activation_pre_in0_0 = std::nullopt) {
```

Comment:
This change to the API looks weird
- Name is not descriptive
- const string is very unconstrained and likely need to be changed to a FusedActivations
- Maybe FusedActivations has to be extended to specify the type to support pre_in0 and other possible options

I want to avoid addition of a new argument like that

Reply:
I just followed the style in `ttnn/cpp/ttnn/operations/matmul.cpp` which use the same string type

Reply:
https://en.wikipedia.org/wiki/Broken_windows_theory

Reply:
@TT-BrianLiu @bbradelTT wdyt?

Reply:
btw, what structure should be used to pass in `FusedActivations` from python?

Reply:
In ttlib, we had `FusibleActivations` I think. Not sure why we have raw strings in ttnn, but we should bind the actual activations themselves and pass them in

Reply:
@TT-BrianLiu yeah that should work, @ayerofieiev-tt if you want no args to be added, how would you like to change the `FusedActivations `

Reply:
yeah I can change to std::optional<UnaryWithParam> input_tensor_a_activation = std::nullopt